src/OpenTelemetry.Instrumentation.GrpcCore/SemanticConventions.cs

```diff
@@ -22,5 +22,11 @@ internal static class SemanticConventions
 
     // Used for unit testing only.
     internal const string AttributeActivityIdentifier = "activityidentifier";
+
+    // https://github.com/open-telemetry/semantic-conventions/blob/main/docs/exceptions/exceptions-spans.md
+    internal const string AttributeExceptionEventName = "exception";
+    internal const string AttributeExceptionType = "exception.type";
+    internal const string AttributeExceptionMessage = "exception.message";
+    internal const string AttributeExceptionStacktrace = "exception.stacktrace";
```

Comment:
All this const are used only in tests file. I am not sure if it worth to put it into production code.

Reply:
Then existing `AttributeActivityIdentifier` const should be also taken into account. I think it is simpler to manage all semantic convetions from one place within a library (including tests). Libraries which use `Shared/SemanticConventions.cs` include many unused consts. BTW probably it is worth to migrate `OpenTelemetry.Instrumentation.GrpcCore` to `Shared/SemanticConventions.cs`. though it lacks "otel.status_code" and "otel.status_description" attributes.

Reply:
Turns out that otel.status_code and otel.status_descrription are used in tests only. As the result of the discussion:
- otel.status_code and otel.status_description should be replaced by `Activity.Status` and `Activity.StatusDescription` in tests
- Use `Shared/SemanticConventions.cs` in `OpenTelemetry.Instrumentation.GrpcCore`

I would like to submit another PR for the suggested changes, @Kielek what do you think ?

Reply:
PRs are more than welcomed :)